### DISCUSSION TITLE
Add toggle to axolotl to prevent attacking squids

### DISCUSSION THREAD

**Discussion by Roan-V** [PurpurMC/Purpur#903] [__Dismissed__] [answered]
Add toggle to axolotl to prevent attacking squids.
https://discord.com/channels/685683385313919172/720375089123688488/788816674555822110

(Toggle to prevent attacking _all_ fish)
https://discord.com/channels/289587909051416579/555469074080202765/788894843161804842

**Comment by granny** [comment, marked as answer]
We've made a data pack that makes axolotls ignore passive aquatic mobs. https://modrinth.com/datapack/purpurpack-axolotls-ignore-passives

If you'd like to configure it more to your liking, some guidelines are below.

A direct link to the current default values for each entity type tag.
https://minecraft.fandom.com/wiki/Tag#Entity_types

The paths & contents of the json files as of `1.18.2`.

`/data/minecraft/tags/entity_types/axolotl_hunt_targets.json`
```json
{
  "replace": true,
  "values": [
    "minecraft:tropical_fish",
    "minecraft:pufferfish",
    "minecraft:salmon",
    "minecraft:cod",
    "minecraft:squid",
    "minecraft:glow_squid"
  ]
}
```

`/data/minecraft/tags/entity_types/axolotl_always_hostiles.json`
```json
{
  "replace": true,
  "values": [
    "minecraft:drowned",
    "minecraft:guardian",
    "minecraft:elder_guardian"
  ]
}
```
